src/CommitmentsV2/SFA.DAS.CommitmentsV2/Models/Cohort.cs

```diff
@@ -658,5 +658,11 @@ public Party Approvals
                 return approvals;
             }
         }
+
+        public void TransferRequestRejectedReturnCohortToEmployer()
+        {
+            CheckIsWithParty(Party.TransferSender);
+            EditStatus = EditStatus.EmployerOnly;
```

Comment:
As we are making a state change, we should start and complete history tracking around the contents of this method (everything after the assertion).

Reply:
added tracking around the update